src/test/starknet.t.sol

```diff
@@ -27,14 +27,16 @@ contract ConfigStarknet {
         uint256 dai_bridge_isOpen;
         uint256 dai_bridge_ceiling;
         uint256 dai_bridge_maxDeposit;
+        uint256 l2_dai_bridge;
```

Comment:
Hey @maciejka , do you want this as bytes32 (what you're assigning when instantiating) or as uint256? IMO I would go with bytes32 here unless there is some context I'm missing here.

Reply:
It represents Starknet address which is a felt (Field Element) whose intepretation is an integer in this case.
It is also declared as uint256 in the bridge contract: https://github.com/makerdao/starknet-dai-bridge/blob/main/contracts/l1/L1DAIBridge.sol#L96 so I would keep it as uint256.

Reply:
Ok makes sense, thanks for explaining.